mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -418,18 +418,24 @@ def _get_records(
             # Frames client expects the aggregators to be a comma-separated string
             agg_funcs = ",".join(agg_funcs)
         table_path = self.tables[table]
-        df = self._frames_client.read(
-            backend=_TSDB_BE,
-            table=table_path,
-            start=start,
-            end=end,
-            columns=columns,
-            filter=filter_query,
-            aggregation_window=interval,
-            aggregators=agg_funcs,
-            step=sliding_window_step,
-            **kwargs,
-        )
+        try:
+            df = self._frames_client.read(
+                backend=_TSDB_BE,
+                table=table_path,
+                start=start,
+                end=end,
+                columns=columns,
+                filter=filter_query,
+                aggregation_window=interval,
+                aggregators=agg_funcs,
+                step=sliding_window_step,
+                **kwargs,
+            )
+        except v3io_frames.ReadError as err:
+            if "No TSDB schema file found" in str(err):
```

Comment:
I believe this can be checked more directly with a V3IO KV looking for the `.#schema` file. What do you think?

Reply:
That would add a redundant, non-transactional I/O operation without good reason.

Reply:
Ok, it just feels a bit sketchy to rely on the error string.

Reply:
Not ideal, but we do it in some places, and we can't rely on a change to frames (because it's versioned as part of the enterprise system).